aksk/pkg/iam/driver/keystone/keystone.go

```diff
@@ -72,6 +76,29 @@ func (iam *KeystoneIam) CreateAkSk(aksk *model.AkSk, req *pb.AkSkCreateRequest)
 		panic(err)
 	}
 
+	// Validate UserId, TenantId .
+	if len(strings.TrimSpace(aksk.ProjectId)) == 0 || len(strings.TrimSpace(aksk.UserId)) == 0 {
+		errMsg := "TenantId or UserId is empty, Please provide valid TenantId and UserId"
+		log.Error(errMsg)
+		return nil, errors.New(errMsg)
+	}
+
+	// Validate if AKSK is being created for a valid User and Tenant.
+	_, err = iam.ValidateId(aksk.ProjectId, req.Token, TENANT_DETAILS)
+
+	if err != nil {
+		errMsg := "TenantId is not valid, Please provide valid TenantId "
```

Comment:
please make the error message start with lowercase. (for all error message)

Reply:
Done